google-chart.html

```diff
@@ -384,6 +404,7 @@
             var chart = chartAndData[0];
             var data = chartAndData[1];
             google.visualization.events.addOneTimeListener(chart, 'ready', function() {
+              this._setImageUri(this.imageURI);
```

Comment:
Yea a computed prop would be good

On Thu, Mar 17, 2016, 11:21 PM Wes Alvaro <notifications@github.com> wrote:

> In google-chart.html
> <https://github.com/GoogleWebComponents/google-chart/pull/87#discussion_r56611886>
> :
>
> > @@ -384,6 +404,7 @@
> >              var chart = chartAndData[0];
> >              var data = chartAndData[1];
> >              google.visualization.events.addOneTimeListener(chart, 'ready', function() {
> > +              this._setImageUri(this.imageURI);
>
> I thought value only set the default (initial) value.
> However, your comment makes me wonder if setting it up as a computed
> property would work.
>
> If a computed property changes in the woods and no-one is observing, does
> it get computed?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleWebComponents/google-chart/pull/87/files/13fab4fd65e80cece450fdab0eb09502978fbd23#r56611886>
>